tests/test_config.cfg

```diff
@@ -1,5 +1,6 @@
 [default]
 endpoint = https://demo.api.lucidtech.ai/v1
-model_names = invoice, receipt
-document_paths = tests/example.jpeg, tests/example.jpeg
-document_mime_types = image/jpeg, image/jpeg
\ No newline at end of file
+model_names = invoice
```

Comment:
Does this mean we will we no longer test for receipts?

Reply:
Fixed